CardPayments/src/main/java/com/paypal/android/cardpayments/CardRequestFactory.kt

```diff
@@ -1,11 +1,11 @@
 package com.paypal.android.cardpayments
 
-import com.paypal.android.cardpayments.api.GetOrderRequest
 import com.paypal.android.corepayments.APIRequest
 import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.api.CoreRequestFactory
 import org.json.JSONObject
 
-internal class CardRequestFactory {
+internal class CardRequestFactory : CoreRequestFactory() {
```

Comment:
question: Are we using inheritance here only for code re-use?

Reply:
yes, thats were the most common "requests" where supposed to live eventually. But we eventually are going to change this architecture